pycbc/events/stat.py

```diff
@@ -1555,6 +1557,12 @@ def rank_stat_coinc(self, s, slide, step, to_shift,
 
         # First get signal PDF logr_s
         stat = {ifo: st for ifo, st in s}
+
+        try:
+            self.curr_mchirp = kwargs['mchirp']
+        except KeyError as e:
+            print(e)
```

Comment:
Either provide a more useful error message, or avoid catching the exception.

Reply:
Because there is no alternative to the 'mchirp' kwarg not existing, you are suggesting just having `self.curr_mchirp = kwargs['mchirp']` without the try except and then if it fails it fails (just like it would with the except anyway)?